tests/conftest.py

```diff
@@ -48,15 +32,14 @@ def mock_clone(_, source, dest):
 
 
 @pytest.fixture
-def tmp_dir():
-    dir_ = tempfile.mkdtemp(prefix="poetry_")
-
-    yield dir_
+def config():  # type: () -> Config
+    with tempfile.NamedTemporaryFile() as f:
+        f.close()
 
-    shutil.rmtree(dir_)
+        return Config(TomlFile(f.name))
 
 
-@pytest.fixture
+@pytest.yield_fixture
```

Comment:
yield_fixture [is deprecated](https://docs.pytest.org/en/latest/yieldfixture.html) so could you switch those back?

Reply:
- switched back to `fixture`